chromium_src/chrome/browser/web_applications/isolated_web_apps/isolated_web_app_update_manager_unittest.cc

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#define ChromeBrowsingDataRemoverDelegate BraveBrowsingDataRemoverDelegate
```

Comment:
I don't understand why this is necessary. testing_profile.cc and isolated_web_app_update_manager_unitttest.cc both use `ChromeBrowsingDataRemoverDelegateFactory` where we have an override to return `BraveBrowsingDataRemoverDelegate` instead of `ChromeBrowsingDataRemoverDelegate`

Reply:
Reverted this change in a2a940dc3df74975e7e685fec4ce2462ea3f2788.

A separate PR has been submitted to fix this override on master https://github.com/brave/brave-core/pull/21034.